src/services/algolia.js

```diff
@@ -0,0 +1,47 @@
+/**
+*
+*  Online store PWA sample.
+*  Copyright 2017 Google Inc. All rights reserved.
+*
+*  Licensed under the Apache License, Version 2.0 (the "License");
+*  you may not use this file except in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*      https://www.apache.org/licenses/LICENSE-2.0
+*
+*  Unless required by applicable law or agreed to in writing, software
+*  distributed under the License is distributed on an "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*  See the License for the specific language governing permissions and
+*  limitations under the License
+*
+*/
+
+const algoliasearch = require('algoliasearch');
+
+const APP_ID = '3S1LH4M84Y';
```

Comment:
Move to config file.

Reply:
I'm not sure this will make it simpler.

I've added an explanation to the Algolia instructions in the README for those that want to use their own data.

Reply:
I'm ok with this for now to get things merged, but configuration mixed with the code like this feels like a code smell to me. If someone forks and runs their own instance then wants to submit back a change from this file then they need to ensure they revert their own config changes before submitting, etc.